plugins/rhs_plugin/rhs_plugin.jsx

```diff
@@ -18,14 +18,21 @@ export default class RhsPlugin extends React.PureComponent {
     }
 
     render() {
+        let header = null;
+        if (this.props.title !== null) {
+            header = (
+                <SearchResultsHeader>
+                    {this.props.title}
+                </SearchResultsHeader>
+            );
```

Comment:
I think we shouldn't couple the title with completely hiding the header. Removing the header removes the option to close and expand the RHS. Not all plugins use the toggle feature, they can just use open the RHS and then they won't have a way to close it. 

If my plugin doesn't want a title but wants to keep the close button?

Reply:
`registerRightHandSidebarComponent(component, '')` and
`registerRightHandSidebarComponent(component)`
Will both work.